src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -39,17 +46,64 @@ public static PackageDocumentationManager Instance
             }
         }
 
+        internal Action<ILogMessage> MessageLogged;
+
+        /// <summary>
+        /// Uses the path manager to set the fallback_doc path for DynamoCore and any Host running
+        /// </summary>
+        /// <param name="pathManager"></param>
+        internal void AddDynamoPaths(IPathManager pathManager)
+        {
+            if (pathManager is null)
+            {
+                return;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.DynamoCoreDirectory))
+            {
+                var coreDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                dynamoCoreFallbackDocPath = coreDir.Exists ? coreDir : null;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
+            {
+                var hostDir = new DirectoryInfo(Path.Combine(pathManager.HostApplicationDirectory, FALLBACK_DOC_DIRECTORY_NAME));
```

Comment:
@sm6srw to answer your question about how useful this is - it actually uses the `DynamoHostPath` property as the root - which can be set using a startupconfiguration - for example, DynamoRevit sets this to the location of the `DynamoRevit.dll` entry point.